lesson07/assignment/a-7/src/components/Jumbotron.vue

```diff
@@ -0,0 +1,30 @@
+<!-- Copyright 2020 Markus Schiffer -->
+<!-- Bootstrap Jumbotron for Landing page. -->
+
+<template>
+    <!-- Uses a Bootstrap jumbotron to display a welcome message. -->
+    <div class="container mt-5">
+        <div class="jumbotron" @mouseover="showMessage = true" @mouseout="showMessage = false">
+            <h1 class="text-primary">Welcome to the landing page!</h1>      
```

Comment:
Remember even for small sentences here we should be using variables from the script, not hardcoded strings.